fmriprep/workflows/anatomical.py

```diff
@@ -6,7 +6,6 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. autofunction:: init_anat_preproc_wf
-.. autofunction:: init_skullstrip_ants_wf
```

Comment:
Can we show the docs for the new skullstrip workflow?

Reply:
I just fixed the docs build and haven't run it myself - IDK.

I'll make sure we plot it.

Reply:
https://3367-53608443-gh.circle-artifacts.com/0/home/circleci/out/docs/workflows.html#brain-extraction-brain-tissue-segmentation-and-spatial-normalization